webrtc-stats.html

```diff
@@ -134,12 +176,16 @@
           Guidelines for design of stats objects
         </h3>
         <p>
-          When introducing a new stats type, the following principles should be followed:
+          When introducing a new stats object, the following principles should be followed:
```

Comment:
I think this was better before. This part of the spec is really about designing new stats *types*, of which there may be multiple instances.